app/lib/utils/parses_cli_style_command_args.rb

```diff
@@ -0,0 +1,25 @@
+require "shellwords"
+
+module Utils
+  class ParsesCliStyleCommandArgs
```

Comment:
@tuxagon I renamed this to `Utils::ParsesCliStyleCommandArgs`. It's not service (e.g. Slack) specific, but is clear about the style of input it expects (something that looks like command line args). Once #26 lands the intent is to add _something_ like:

```ruby
def command_params
  ActionController::StrongParameters.new(Utils::ParsesCliStyleCommandArgs(request.parameters[:text]).args)
end

def subcommand
  Utils::ParsesCliStyleCommandArgs(request.parameters[:text]).subcommand
end
```
so chatops controllers can just use a flavor of params as other controllers would. If we ever include anything other than slack we can refactor accordingly.

Reply:
I think that makes sense!